.github/workflows/test-library.yml

```diff
@@ -340,7 +340,7 @@ jobs:
         - build-docs
         - doctest-docs
         - linkcheck-docs
-        - spellcheck-docs
+        # - spellcheck-docs
```

Comment:
@abhinavsingh

This is the upstream issue: https://github.com/sphinx-contrib/spelling/issues/227.
I didn't mean disabling the entire check, just flipping the PyPI lookup setting: https://github.com/ansible/awx-plugins/commit/842fb8d626e2ea41e4371727e289547d20c75365.